aip/general/0009.md

```diff
@@ -147,7 +151,16 @@ An API that operates across a network of computers. Network APIs communicate
 using network protocols including HTTP, and are frequently produced by
 organizations separate from those that consume them.
 
-[IaC]: #iac
+### Single Owner Fields
```

Comment:
I don't think single-owner fields should be a glossary term. See https://github.com/aip-dev/google.aip.dev/pull/1144/files#r1272461411.

at minimum the guidance needs to be moved to an AIP that's not the glossary.

Reply:
Removed from this PR, will add back to the 129 PR